### PR TITLE
Content Blocks import

### PR DESCRIPTION
**Pull request by aapomm**
- Added method `create_content_blocks` to content service 
- Add the ability to pass a `default_user_id` to the importer that you can use to fill in the required user associations when importing data.

### CODE REVIEW

File: lib/dradis/plugins/upload/importer.rb [hunk 1]
```diff
@@ -5,14 +5,23 @@ module Dradis
...
+          :template_service
+        )
 
         def initialize(args={})
           @options = args
 
           @logger  = args.fetch(:logger, Rails.logger)
           @plugin  = args[:plugin] || default_plugin
           @project = args.key?(:project_id) ? Project.find(args[:project_id]) : nil
+          @default_user_id = args[:default_user_id] || -1
```
Comment (dormi) [inline lib/dradis/plugins/upload/importer.rb]: `args.fetch(:default_user_id, -1)` ?

**Comment by etdsoft** [comment]
None of this belongs to CE or `dradis-plugins` at most a dummy / no-opt method.

**Comment by dormi** [comment]
@etdsoft let's move content blocks and boards out of this plugin content service in the future?

File: lib/dradis/plugins/content_service/nodes.rb [hunk 2]
```diff
@@ -74,7 +74,7 @@ def nodes_from_evidence
     #
     # Returns and Array with a unique collection of Nodes.
     def nodes_from_properties
-      Node.user_nodes.where('properties IS NOT NULL AND properties != \'{}\'')
+      Node.user_nodes.where("type_id <> #{Node::Types::CONTENTLIB} AND properties IS NOT NULL AND properties != '{}'")
```
Comment (etdsoft) [inline lib/dradis/plugins/content_service/nodes.rb]: The `:user_nodes` scope already excludes `CONTENTLIB`

**Comment by etdsoft** [comment]
> @etdsoft let's move content blocks and boards out of this plugin content service in the future?

I think this is fine as it is.

File: lib/dradis/plugins/content_service/nodes.rb (hunk 2 above)
Comment (dormi) [reply lib/dradis/plugins/content_service/nodes.rb]: I think `user_nodes` work, this change was an error. Reverted.

**Pull request merged**